sprintvotingapp/sprint/views.py

```diff
@@ -0,0 +1,536 @@
+import logging
+from collections import Counter
+from itertools import count
+from django.db import transaction
+from .models import Sprint, Parameter, Votes
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from .serializer import SprintSerializer, ParamSerializer, VoteSerializer
+from rest_framework.exceptions import ValidationError
+from .utils import InsertionError, AlreadyPresentException, verify_token
+from user.models import User
+
+logging.basicConfig(filename="sprint.log", filemode="w")
+
+
+class SprintC(APIView):
+    @verify_token
+    def post(self, request):
+        """
+        For Adding sprint Cycle
+        :param request:
+        :return:Response
+        """
+        try:
+            user = User.objects.get(id=request.data.get("user_id"))
+
+            if user.is_superuser:
```

Comment:
Code to check superuser is being repeated, can be handled in a single place maybe? Can weuse DRF permission classes or create a  permissions decorator like token_verify to minimise code repetition?

Reply:
Added a is_superuser decorator